app/management/commands/update_db.py

```diff
@@ -1,16 +1,18 @@
 from django.core.management.base import BaseCommand
 
-from app.src.update_db import update_all, update_all_for_region
+from app.src.update_db import update_all, update_all_for_region, update_gm_for_region_legacy
 
 
 class Command(BaseCommand):
     help = 'Pulls ladder data from the Blizzard API into our database.'
 
     def add_arguments(self, parser):
         parser.add_argument('--region', help='Region to update')
+        parser.add_argument('--legacy', action='store_true', help='Whether to use legacy method to update ladder')
```

Comment:
what's up with `action='store_true'`?

Reply:
`action='store_true'` makes it so you don't need to specify a parameter after `--legacy`. If you include `--legacy` then the option will be set to true, if you don't include it then the option will be set to false.